sdk/identity/identity/review/identity.api.md

```diff
@@ -218,7 +213,7 @@ export const logger: AzureLogger;
 export class ManagedIdentityCredential implements TokenCredential {
     constructor(clientId: string, options?: TokenCredentialOptions);
     constructor(options?: TokenCredentialOptions);
-    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
```

Comment:
I'm making this change because the pipeline thinks it shouldn't be happening.

```suggestion
    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
```